README.md

```diff
@@ -30,6 +24,25 @@
         ></a>
 </div>
 <br>
+<div align="center">
+   <img 
+        src="https://img.shields.io/badge/python%20version-3.9-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.9" 
+    />
+   <img 
+        src="https://img.shields.io/badge/python%20version-3.10-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.10" 
+    />
+    <img 
+        src="https://img.shields.io/badge/python%20version-3.11-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.11" 
+    />
+        <img 
+        src="https://img.shields.io/badge/python%20version-3.12-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.12" 
+    />
```

Comment:
```suggestion
```

Reply:
I have pushed the fix for above feedback
<img width="1014" alt="image" src="https://github.com/user-attachments/assets/40a75fe4-dfc5-4057-8fe5-f678a324817b">